test/authentication_tests.coffee

```diff
@@ -73,10 +77,11 @@ Meteor.startup ->
         test.equal response.status, 'success'
         next()
 
-    it 'should remove the logout token after logging out', (test, next) ->
+    it 'should remove the logout token after logging out and should respond after 500 msec', (test, next) ->
```

Comment:
I think that the unit test should be self contained (black box testing). In other words, each unit test should ensure that a requirement is met, without knowledge of the implementation.

We have two requirements:

- When I log in with bad credentials, the system should respond with a 401, no sooner than 500 msec.
- When I attempt to access a restricted resource without authorization, the system should respond with a 401, no sooner than 500 msec.

In the future, the implementation may change and someone may implement the delay for bad credentials, but forget to delay for unauthorized access.
Since security is a risky area and exposes protected content, we should be extra careful about risks.